rsocket-core/src/main/java/io/rsocket/internal/UnboundedProcessor.java

```diff
@@ -40,6 +40,7 @@
  *
  * @param <T> the input and output type
  */
+@Deprecated
```

Comment:
This could be made package private right away. It's an internal class anyway. However see related comments on `BaseDuplexConnection` as well.

Reply:
Figured out that this Processor is needed in a few more places, thus removed this annotation for now and kept this impl public